app/services/uriPreview.server.ts

```diff
@@ -71,9 +77,7 @@ export async function getUriPreview(uri: string): Promise<PreviewResult> {
     return createPreviewJson(url.href, jsonBody);
   }
 
-  const peekalinkResult = await getPeekalink(url.href);
-
-  return peekalinkResult;
+  return await getOpenGraphNinja(`${url.hostname}${url.pathname}${url.search}`);
```

Comment:
Is there a reason this isn't using `url.href` like the previous code was?

Reply:
@ericallam This was a mistake on my part. I misread the documentation for OpenGraph. I thought it said we needed to use a URL _without_ the protocol, but it's saying you can use _either_. Good catch! I've gone ahead and switched it back to using the `url.href` 👍.

<img width="605" alt="image" src="https://github.com/triggerdotdev/jsonhero-web/assets/11434879/9e17e1d8-36e8-45f8-9e2b-e0103fd0c7c3">